### PR TITLE
Merged RemoveDeadZeroAndOneOperationRector and RemoveZeroAndOneBinarRector

### PR DESCRIPTION
**Pull request by jeroensmit**
Probably a rename gone wrong once

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Could you rebase on `master`M

File: docs/NodesOverview.md-E [hunk 1]
```diff
@@ -0,0 +1,1114 @@
+# Node Overview
```
Comment (TomasVotruba) [inline docs/NodesOverview.md-E]: I guess this file should be removed

Comment (jeroensmit) [reply docs/NodesOverview.md-E]: ah, yeah. You have any idea what is causing those files to appear?

Comment (TomasVotruba) [reply docs/NodesOverview.md-E]: Maybe this https://github.com/rectorphp/rector/commit/337cbd3ad470833f909b1434f49bc326be93121e ?

Comment (jeroensmit) [reply docs/NodesOverview.md-E]: the adding was accidental

found the reason the files appeared:
https://github.com/rectorphp/rector/pull/2318

**Comment by TomasVotruba** [comment]
CI needs fixing

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**